index.js

```diff
@@ -26,6 +26,12 @@ function middleware (apiPath, api, options) {
 
   const router = new Router()
 
+  router.use((req, res, next) => {
+    req.sparql = client
+    if (next) {
+      next()
+    }
+  })
```

Comment:
I do not particularly really like this stuffed here inline. Would you have me put it somewhere else?

Reply:
It's OK there now. Later we should separate the complete SPARQL client stuff into a middleware, so creating the client and attaching it to the request is done only in the middleware. Then other middlewares must use only the SPARQL client from the request and it's no longer required to pass it around.